src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubOrgWebHook.java

```diff
@@ -48,20 +48,11 @@
     private static final Logger LOGGER = Logger.getLogger(GitHubOrgWebHook.class.getName());
     private static final List<GHEvent> EVENTS = Arrays.asList(GHEvent.REPOSITORY, GHEvent.PUSH, GHEvent.PULL_REQUEST, GHEvent.PULL_REQUEST_REVIEW_COMMENT);
 
-    /**
-     * Verify if exists a webhook by its URL.
-     */
-    private static boolean existsHook(GHOrganization org, String url) throws IOException {
-        for (GHHook hook : org.getHooks()) {
-            if (hook.getConfig().get("url").equals(url)) {
-                return true;
-            }
-        }
-        return false;
-    }
-
     public static void register(GitHub hub, String orgName) throws IOException {
-        String rootUrl = Jenkins.getActiveInstance().getRootUrl();
+        String rootUrl = System.getProperty("jenkins.hook.url");
```

Comment:
@jglick  Does that automatically convert from the `JENKINS_HOOK_URL` env value as well?